libreport.spec.in

```diff
@@ -54,6 +57,7 @@ BuildRequires: xz
 BuildRequires: lz4
 Requires: libreport-filesystem = %{version}-%{release}
 Requires: satyr >= 0.18
+Requires: glib2 >= %{glib_ver}
```

Comment:
@jfilak rpmbuild automatically add just libglib-2.0.so.0()(64bit) without the version. And as stated there are unfortunatelly older version of glib2 which are not compatible with libreport.